larq/callbacks.py

```diff
@@ -1,6 +1,9 @@
 import tensorflow as tf
 import numpy as np
 
+from larq.optimizers import Bop
+from tensorflow.python.keras import backend as K
```

Comment:
In general, we should try not to import from `tensorflow.python` since this is not a public API and might change.

Reply:
I replaced the K with tf.keras.backend everywhere in the code.

Reply:
> I replaced the K with tf.keras.backend everywhere in the code.

Looks like it's still imported though

Reply:
I haven't push the changes yet.